app/javascript/components/widgets/traffic/panel.jsx

```diff
@@ -89,13 +90,13 @@ const Traffic = () => (
         <>
           <Row>
             <Column>
-              {take(3, firstColumn).map(cam => (
-                <TrafficCam {...cam} />
+              {take(3, firstColumn).map(({ id, ...cam }) => (
+                <TrafficCam key={id} {...cam} />
```

Comment:
squash w/ 1st commit?

Reply:
Ah ok. My intent here was the key change was a bugfix that was simplified with the preceding refactor. Maybe that's too sub-atomic. I'll have a look at the squashed result.

Reply:
Maybe I misunderstood the changes -- it looked like there was a `key` being applied originally https://github.com/mojotech/helios2/pull/161/files#diff-9a23cb58dd0b34ddf1471cd7d9aff51eL76

Reply:
Finally got back to this. `<div key={cam}>` was the issue since an object was being passed to key which resulted in the react warnings. I'll make that more clear.